src/renderer/contexts/ExecutionContext.tsx

```diff
@@ -266,10 +268,14 @@ export const ExecutionProvider = memo(({ children }: React.PropsWithChildren<{}>
                 });
             }
         } catch (err: unknown) {
-            sendAlert({
-                type: AlertType.ERROR,
-                message: `An unexpected error occurred: ${String(err)}`,
-            });
+            if (err instanceof DOMException && err.name === 'AbortError') {
+                // We could maybe send a success message here
```

Comment:
If we aborted successfully, then the user will see the control buttons in the UI change. No need for any messages here.